docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,66 +3,66 @@
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":WanderlustParser" as WanderlustParser LOGIC_COLOR
 participant ":DeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:DeleteCommand" as DeleteCommand LOGIC_COLOR
+participant "d:DeleteActivityCommand" as DeleteActivityCommand LOGIC_COLOR
 participant ":CommandResult" as CommandResult LOGIC_COLOR
 end box
 
 box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("delete 1")
+[-> LogicManager : execute("delete -activity 1")
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("delete 1")
-activate AddressBookParser
+LogicManager -> WanderlustParser : parseCommand("delete -activity 1")
+activate WanderlustParser
 
 create DeleteCommandParser
-AddressBookParser -> DeleteCommandParser
+WanderlustParser -> DeleteCommandParser
 activate DeleteCommandParser
```

Comment:
DeleteCommandParser is Called WanderlustDeleteCommandParser for all

Reply:
Actually I changed all to without all the Wanderlust in the front. Tsk didn't read my telegram message. YOU SHOULD CHANGED!!! NOT ME!!! HAHAHAHA

Reply:
OK MY BAD